### PR TITLE
Display githash used for run

### PR DESCRIPTION
**Pull request by cheng-kevin**

### CODE REVIEW

**Comment by ryanjulian** [comment]
Can you show a screenshot or demo?

**Comment by cheng-kevin** [comment]
Screen Shot 2019-05-14 at 12 15 08 PM

File: docs/index.html [hunk 1]
```diff
@@ -15,6 +15,7 @@
                 <h1>Garage Benchmarking</h1>
                 <br/>
                 <p> Time last updated: <small id="time"></small> </p>
+                <p> Ran from commit: <a id="githash"> link </a> </p>
```
Comment (ryanjulian) [inline docs/index.html]: can you actually use the hash itself for the link? it is useful to be able to easily copy it to check out that revision from a terminal.

Comment (cheng-kevin) [reply docs/index.html]: Screen Shot 2019-05-14 at 2 55 34 PM

**Pull request merged**